packages/wallets/src/components/AppCardBadge/AppCardBadge.scss

```diff
@@ -5,7 +5,8 @@
     justify-content: center;
     align-items: center;
     height: 2.2rem;
-    border-radius: 0 0.8rem;
+    border-start-end-radius: 0.8rem;
+    border-end-start-radius: 0.8rem;
```

Comment:
What's the difference between these two?

Reply:
top-right and bottom-left borders in ltr or top-left and bottom-right borders in rtl. both are needed